src/wilcoxon.jl

```diff
@@ -163,6 +163,26 @@ function StatsAPI.pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
+function psignrank(W::T, n::T, lower_tail::Bool) where {T<:Integer}
```

Comment:
IMO we should just keep it simple for now, in HypothesisTests we only need
```suggestion
function psignrank(W::Float64, n::Int, lower_tail::Bool)
```
or
```suggestion
function psignrank(W::Int, n::Int, lower_tail::Bool)
```
it seems, depending on whether we round above or inside of `psignrank`.